app/services/taxon_base_path_structure_check.rb

```diff
@@ -0,0 +1,71 @@
+class TaxonBasePathStructureCheck
+  LEVEL_ONE_URL_REGEX = %r{^\/([A-z0-9\-]+)$}
+  PATH_COMPONENTS_REGEX = %r{^\/(?<prefix>[A-z0-9\-]+)(\/(?<slug>[A-z0-9\-]+))?$}
+
+  attr_reader :path_validation_output, :invalid_taxons
+
+  def initialize(parent_taxons:)
+    @parent_taxons = parent_taxons
```

Comment:
It might be clearer to change `parent_taxons` to `level_one_taxons` here, as that's what's expected?

Reply:
Yep, good point. I think I was half way through renaming :)